src/manifest/mod.rs

```diff
@@ -169,7 +148,53 @@ impl CargoManifest {
             files: files,
             main: js_file,
             types: dts_file,
+        })
+    }
+
+    fn into_esmodules(mut self, scope: &Option<String>, disable_dts: bool) -> NpmPackage {
+        let filename = self.package.name.replace("-", "_");
+        let wasm_file = format!("{}_bg.wasm", filename);
+        let js_file = format!("{}.js", filename);
+
+        let dts_file = if disable_dts == true {
+            None
+        } else {
+            Some(format!("{}.d.ts", filename))
+        };
+
+        if let Some(s) = scope {
```

Comment:
This scope handling logic looks duplicated between this and the above?

Reply:
this is all from master but yeah you are right i think, file an issue?